nixos/tests/k3s/multi-node.nix

```diff
@@ -138,12 +138,15 @@ import ../make-test-python.nix ({ pkgs, lib, ... }:
           m.start()
           m.wait_for_unit("k3s")
 
+      is_aarch64 = "${toString pkgs.stdenv.isAarch64}" == "1"
+
       # wait for the agent to show up
       server.wait_until_succeeds("k3s kubectl get node agent")
 
       for m in machines:
-      '' # Fix-Me: Tests fail for 'aarch64-linux' as: "CONFIG_CGROUP_FREEZER: missing (fail)"
-      + lib.optionalString (!pkgs.stdenv.isAarch64) ''m.succeed("k3s check-config")'' + ''
+          # Fix-Me: Tests fail for 'aarch64-linux' as: "CONFIG_CGROUP_FREEZER: missing (fail)"
```

Comment:
I'm wondering if there's any reason we don't enable this in the kernel config on aarch64?

Reply:
Seems the right solution. Someone has to make it work. And provide a patch for that.

Reply:
Unfortunately I'm not super familiar with the state of Linux on arm, so I can't take this on. It does seem like rebuilding my arm box with the flag flipped works "fine" and k3s appears happy, but that's not a call I'm willing to make for everyone else :sweat_smile:

Reply:
Will check. Patch was in previous version.